### ISSUE TITLE
Aggregate features promoted within the experience upon load

### ISSUE THREAD

**Issue by alexbfree** [hestiaAI/hestialabs-experiences#1129]
This comes from a request from @foucault-dumas, first raised [here](https://github.com/hestiaAI/clients/issues/54) and moved over to this separate ticket.

From Charles:

> this sentence from POD [this week weekly meeting](https://kdrive.infomaniak.com/app/office/193995/192906): "start the experiences with an aggregated view --> to showcase the aggregation results"

Alex response:

> @foucault-dumas is that just a note for future content to be written once CMS feature is available, or is there some dev implication here? (In other words, is there any CMS-editor feature this would require that isn't already documented above?)

Charles' answer:

> I don't know where it belongs, maybe it's in a non-existing-yet-issue about the fact that when one enters any data exp, the first thing (s)he sees is the huge potential of the aggregation. I think that happens just after [that](https://github.com/hestiaAI/clients/issues/54#issuecomment-1274684888), in a first tab (if tabs are still used tomorrow). Any comment/remark about it? Would you create this issue?

Alex response

> Thanks for your reply. I think it is a different issue, possibly best addressed after #24 and #54. I will create an issue (that's this issue) and move these comments across.

**Comment by alexbfree** [comment]
@foucault-dumas I will leave this ticket here so that we can refine the requirements and also you can set a priority if you like.

Questions:
- What is the priority?
- What exactly you would want to see on screen. 
- Which experiences is this for? 
- How does this mesh into the four-part (now two part) grid design in https://github.com/hestiaAI/clients/issues/54#issuecomment-1274684888 ?
